packages/core/interfaces/index.ts

```diff
@@ -133,12 +128,21 @@ export interface UiCartProduct {
   id: string
   image: string
   price: UiCartProductPrice
-  configuration: UiCartProductConfiguration[]
+  configuration: AgnosticProductAttribute[]
   qty: string
 }
 
 export interface AgnosticProductAttribute {
-  name: string,
+  name: string
   value: string | Object
   label: string
 }
+
+export interface AgnosticProductOption {
+  value: string | object
+  label: string
+}
+
+export type AgnosticProductOptions<A extends string> = Record<A, AgnosticProductOption>
```

Comment:
imho it's redundant, why not to just keep `AgnosticProductOption` as it's just an array of that ?